assets/js/custom.js

```diff
@@ -106,3 +106,24 @@ document.addEventListener("click", function(e) {
     closeOpenedMenusExcept();
   }
 });
+
+function update_switch_theme_button() {
+    current_theme = document.documentElement.dataset.mode;
```

Comment:
```suggestion
    current_theme = document.documentElement.dataset.mode;
    if (!current_theme) return;
```

The `document.documentElement.dataset.mode` part here returns `null` for Doxygen docs. So we should probably add an early `return` statement to prevent any errors from occurring.